src/App.tsx

```diff
@@ -10,33 +11,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const onTabSelected = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const id = event.currentTarget.hash.slice(1);
+
+    setSelectedTabId(id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is Tab
+        {' '}
+        {selectedTabId.at(-1)}
```

Comment:
You have a title in each tab for it, so use it

Reply:
done